guide/how-to.md

```diff
@@ -1,87 +1,153 @@
+
 
-![biped.png](images/biped.png)
+### contact\_points.wbt
 
-%end
+**Keywords**: Physics plugin, contact points
+
+![contact_points.png](images/contact_points.png) In this example, a box falls until it reaches the ground.
+A physics plugin (cf. `WorldInfo.physics`) gets information on the contact points and displays the reaction forces on the `Console` and as red lines.
+
+
+### cylinder\_stack.wbt
+
+**Keywords**: Supervisor, contact points, cylinder collisions
+
+![cylinder_stack.png](images/cylinder_stack.png) In this example, a stack of cylinders collapses.
+A Supervisor gets information on the contact points and displays the reaction forces in the `Console`.
 
-In this example, a biped robot stands up while his head rotates. After a few
-seconds, all the motors are turned off and the robot collapses. This example
-illustrates how to build a simple articulated robot and also how to turn off
-motor power.
 
 ### force\_control.wbt
 
 **Keywords**: Force control, linear motor, spring and damper
 
-%figure "force_control"
+![force_control.png](images/force_control.png) This world shows two boxes connected by a `LinearMotor`.
+Here, the purpose is to demonstrate the usage of the `wb_motor_set_force()` function to control a `LinearMotor` with a user specified force.
+In this example, `wb_motor_set_force()` is used to simulate the effect of a spring and a damper between the two boxes. When the simulation starts, the motor force is used to move the boxes apart.
+Then, the motor force is turned off and the boxes oscillate for a while, according to the spring and damping equations programmed in the controller.
+
+
+### four\_wheels.wbt
 
-![force_control.png](images/force_control.png)
+**Keywords**: four-wheeled frame
 
-%end
+![four_wheels.png](images/four_wheels.png) This example shows three models of four-wheeled vehicles.
+In the first layout, the four motorized wheels are positioned circularly, as could be done for omnidirectional-wheeled robots.
+In the second layout, the four motorized wheels are oriented in the same direction, as could be done for a robot with tracks.
+In the third layout, a simple [Ackermann steering geometry](https://en.wikipedia.org/wiki/Ackermann_steering_geometry) is shown.
+Note that more completed Ackermann steering geometry can be achieved using the `AckermannVehicle.proto` model, and the `vehicle` library (cf. the [automobile book](https://www.cyberbotics.com/doc/automobile/index)).
+
+
+### gui\_tracker.wbt
+
+**Keywords**: GUI, custom Qt-based robot window
+
+![gui_tracker.png](images/gui_tracker.png) This example shows the use of the deprecated Qt-based system to create custom windows.
+A Supervisor is getting the position of five e-pucks moving randomly, and this information is sent to the custom robot window plugin called `tracking window`.
+This window is based on Qt (embedded in Webots) and the `Qt utils` helper library.
```

Comment:
We should update this example to use the HTML robot window instead...

Reply:
I agree; I created a hand-written list of samples which should be completed or added, this one is probably the most important.